zerver/tests/test_import_export.py

```diff
@@ -296,6 +297,58 @@ def _setup_export_files(self, realm: Realm) -> Tuple[str, str, str, bytes]:
+            'INFO:root:Successfully imported <class \'zerver.models.UserHotspot\'> from zerver_userhotspot.',
+            'INFO:root:Successfully imported <class \'zerver.models.MutedTopic\'> from zerver_mutedtopic.',
+            'INFO:root:Successfully imported <class \'zerver.models.Service\'> from zerver_service.',
+            'INFO:root:Successfully imported <class \'zerver.models.UserGroup\'> from zerver_usergroup.',
+            'INFO:root:Successfully imported <class \'zerver.models.UserGroupMembership\'> from zerver_usergroupmembership.',
+            'INFO:root:Successfully imported <class \'zerver.models.BotStorageData\'> from zerver_botstoragedata.',
+            'INFO:root:Successfully imported <class \'zerver.models.BotConfigData\'> from zerver_botconfigdata.',
+            'INFO:root:Successfully imported <class \'zerver.models.UserPresence\'> from zerver_userpresence.',
+            'INFO:root:Successfully imported <class \'zerver.models.UserActivity\'> from zerver_useractivity.',
+            'INFO:root:Successfully imported <class \'zerver.models.UserActivityInterval\'> from zerver_useractivityinterval.',
+            'INFO:root:Successfully imported <class \'zerver.models.CustomProfileField\'> from zerver_customprofilefield.',
+            'INFO:root:Successfully imported <class \'zerver.models.CustomProfileFieldValue\'> from zerver_customprofilefieldvalue.',
+            'INFO:root:Importing avatars',
+            'INFO:root:Importing uploaded files',
+            'INFO:root:Importing emojis',
+            'INFO:root:Importing realm icons and logos',
+            'INFO:root:Successfully rendered Markdown for message batch',
+            'INFO:root:Successfully imported <class \'zerver.models.Message\'> from zerver_message.',
+            'INFO:root:Successfully imported <class \'zerver.models.UserMessage\'> from zerver_usermessage[1].',
+            'INFO:root:Successfully imported <class \'zerver.models.Reaction\'> from zerver_reaction.',
+            'INFO:root:Successfully imported <class \'zerver.models.Attachment\'> from zerver_attachment.',
+            'INFO:root:Successfully imported M2M table zerver_attachment_messages',
+            'INFO:root:Successfully imported <class \'analytics.models.RealmCount\'> from analytics_realmcount.',
+            'INFO:root:Successfully imported <class \'analytics.models.UserCount\'> from analytics_usercount.',
+            'INFO:root:Successfully imported <class \'analytics.models.StreamCount\'> from analytics_streamcount.'
+        ])
+        self.assertEqual(logs[31][:33] + logs[31][-33:], 'INFO:root:Importing message dump /test-export/messages-000001.json')
+        self.assertEqual(logs[36][:41] + logs[36][-28:], 'INFO:root:Importing attachment data from /test-export/attachment.json')
+        self.assertEqual(logs[39][:40] + logs[39][-27:], 'INFO:root:Importing analytics data from /test-export/analytics.json')
```

Comment:
what are the numbers here? 🤔

Reply:
These logs included had system dependent file path which would change if tests are run on any other system or even using circle-ci. For example one such log was:
`INFO:root:Importing realm data from /home/purushottam/development/zulip/var/e1d49ad1-aed1-4c1d-81ef-a45d647296b5/test-backend/run_1514398/worker_0/test-export/realm.json`
For such logs I have used string slicing to assert only important and constant part of logs in order to maintain consistency of tests. Also some logs were simple meaning they did not contain any data that was variable so I have simply asserted them. 
This was my approach for asserting such variable logs. If you suggest some other way I will be happy to implement.